### DISCUSSION TITLE
[WIKI] Theme gallery

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#3050] [Show and tell]
Hello, @zadam .

I had a look at the [User provided themes](https://github.com/zadam/trilium/wiki/Themes#user-provided-themes) page and thought it would be a nice idea to show thumbnails for each of the themes. So I've created a separate page to list the themes and liked the previous section to this new page.

Before:

![image](https://user-images.githubusercontent.com/21236836/183521620-53c04c0e-0b50-4f57-b0cf-e66a76876a79.png)

After:

![image](https://user-images.githubusercontent.com/21236836/183521562-2c1233c1-7561-4097-af41-342b49e8d100.png)

You can find the page on [my fork](https://github.com/eliandoran/trilium/wiki/Theme-gallery). I would like to add it to the Wiki, but I do not have write access.

What do you think about it?

**Comment by zadam** [comment]
Hi, that's nice. I added you as a collaborator which should allow you to edit wiki.

**Reply by eliandoran** [reply]
@zadam , that's great. It's live: https://github.com/zadam/trilium/wiki/Theme-Gallery :)
